gsd/hoomd.py

```diff
@@ -530,6 +531,19 @@ def __len__(self):
         """ The number of frames in the trajectory. """
         return self.file.nframes;
 
+    def __getstate__(self):
+        '''Checks if in read mode and removes file handle from __dict__'''
```

Comment:
This docstring appears out of date since `__dict__` is not updated.

Reply:
It was.